pkg/state/state.go

```diff
@@ -629,7 +629,7 @@ func (s *stateManager) BlockVRF(blockHeader *proto.BlockHeader, height proto.Hei
 	if blockHeader.Version < proto.ProtobufBlockVersion {
 		return nil, nil
 	}
-	pos := &consensus.FairPosCalculatorV2{}
+	pos := consensus.FairPosCalculatorV2
```

Comment:
Are you sure that here should be hardcoded `FairPosCalculatorV2`? I think you should use `consensus.NewFairPosCalculator` here to create appropriate PoS calculator.